.env

```diff
@@ -0,0 +1,6 @@
+TZ=your_time_zone
+GUI_PWD=your_gui_password
+WEBUI_PWD=your_webui_passwrod
+DOCKERDIR=your_docker_path
+INCOMING=your_incoming_path
+TEMP=your_temp_path
```

Comment:
What about adding this all to the original README? So everyone sees this configuration you proposed directly in the landing page of the repo.